<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_login</class>
 <widget class="QDialog" name="Dialog_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Подключение к БД</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grB_connect">
     <property name="title">
      <string>Подключение</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_hostname">
        <property name="text">
         <string>Имя хоста:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_hostname">
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spB_port">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
        <property name="value">
         <number>5432</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_databaseName">
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_port">
        <property name="text">
         <string>Порт:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_databaseName">
        <property name="text">
         <string>Имя БД:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grB_login">
     <property name="title">
      <string>Вход</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_login">
        <property name="text">
         <string>Логин:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_login">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_pass">
        <property name="maxLength">
         <number>30</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_pass">
        <property name="text">
         <string>Пароль:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bbox_OkCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bbox_OkCancel</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_login</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbox_OkCancel</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_login</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
